### ISSUE TITLE
Add contributors section

### ISSUE THREAD

**Issue by Soumya-Kushwaha** [The-Robotics-Forum/flutter-ui-templates#57]
Assign me plz!

**Comment by SiddhantPawar03** [comment]
> Assign me plz!

Are you HSOC'22 Participant?
Also please refer [flutter-ui-templates](https://github.com/The-Robotics-Forum/flutter-ui-templates)/[issue_template](https://github.com/The-Robotics-Forum/flutter-ui-templates/tree/main/issue_template)/feature_request.md for creating new issue.

**Comment by Soumya-Kushwaha** [comment]
Yes I am a HSOC '22 participant

This edit would enhance the readme and display all the contributors on the main readme

**Comment by SiddhantPawar03** [comment]
> Yes I am a HSOC '22 participant
> 
> This edit would enhance the readme and display all the contributors on the main readme

If you are able to update the whole readme then you may proceed as adding just contributors section might not be able to assign labels.
Also the level of labels would be assign once readme is done and level of readme.
For better clarification you may use feature_request.md (issues template).
